### ISSUE TITLE
Error: Too many requests to sparql

### ISSUE THREAD

**Issue by themonk911** [GoogleCloudPlatform/covid-19-open-data#258]
```
{
  "insertId": "4prfvhl8rre1qyryb",
  "jsonPayload": {
    "logname": "WikidataDataSource",
    "classname": "WikidataDataSource",
    "message": "<html>\n<head>\n<meta http-equiv=\"Content-Type\" content=\"text/html;charset=utf-8\"/>\n<title>Error 429 Too Many Requests - Please retry in 0 seconds.</title>\n</head>\n<body><h2>HTTP ERROR 429</h2>\n<p>Problem accessing /bigdata/namespace/wdq/sparql. Reason:\n<pre>    Too Many Requests - Please retry in 0 seconds.</pre></p><hr><a href=\"http://eclipse.org/jetty\">Powered by Jetty:// 9.4.12.v20180830</a><hr/>\n\n</body>\n</html>\n",
    "timestamp": "2020-10-22T11:52:34.9476"
  },

  "timestamp": "2020-10-22T11:52:34Z",

  "receiveTimestamp": "2020-10-22T11:52:36.945038121Z"
}
```

**Comment by themonk911** [comment]
```
<html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
<title>Error 429 Too Many Requests - Please retry in 0 seconds.</title>
</head>
<body><h2>HTTP ERROR 429</h2>
<p>Problem accessing /bigdata/namespace/wdq/sparql. Reason:
<pre>    Too Many Requests - Please retry in 0 seconds.</pre></p><hr><a href="http://eclipse.org/jetty">Powered by Jetty:// 9.4.12.v20180830</a><hr/>

</body>
</html>
```

**Comment by owahltinez** [comment]
Fixed via #309
